src/applications/personalization/profile/components/edit/Edit.jsx

```diff
@@ -78,14 +78,18 @@ export const Edit = () => {
     hasVAPServiceConnectionError(state),
   );
 
-  const fieldData = useSelector(state =>
-    selectVAPContactInfoField(state, fieldInfo?.fieldName),
-  );
-
   const hasUnsavedEdits = useSelector(
     state => state.vapService.hasUnsavedEdits,
   );
 
+  // used to make sure the modal 'editing' state for the field is present
+  // it should match the fieldName that is in the URL query param
+  const modal = useSelector(state => state?.vapService.modal);
```

Comment:
`vapService` will always be present since it is present in the initial state of the reducer of this store, but modal is initially set explicitly to `null`